ci/run_playbooks_logging.yml

```diff
@@ -0,0 +1,98 @@
+---
+- name: "Run logging functional test playbooks"
+  hosts:  "{{ cifmw_target_hook_host | default('localhost')  }}"
+  gather_facts: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars_files:
+    - vars/common.yml
+  tasks:
+    - name: "Create log dir"
+      ansible.builtin.file:
+        path: "{{ logs_dir }}"
+        state: directory
+        mode: "0755"
+
+    - name: "Set dynamic KUBECONFIG and PATH vars"
+      copy:
+        content: |
+          cifmw_openshift_kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+          cifmw_path: "{{ cifmw_path }}"
+          vars_dir: "{{ fvt_dir }}/ci/vars"
+        dest: "{{ env_vars_file }}"
+
+    - name: Include vars from the extra_vars files
+      ansible.builtin.include_vars:
+        dir: "{{ cifmw_basedir }}/artifacts/parameters"
+
+    # Prepare the env
+    - name: Setup the required resources before running the logging functional test
+      block:
+          # TODO: Look at replacing this with something NOT from install_yamls.
+        - community.general.make:
+            chdir: '{{ ansible_env.HOME }}/{{ zuul.projects["github.com/openstack-k8s-operators/install_yamls"].src_dir }}/devsetup'
+            target: edpm_deploy_instance
+          tags:
+            - setup
+
+        - name: Patch the openstackversions to use the master containers for aodh and heat
+          ansible.builtin.shell:
+            cmd: |
+              oc patch openstackversions openstack-galera-network-isolation --type merge --patch-file ci/patch-openstack-versions.yaml
+            chdir: "{{ fvt_dir }}"
+          when: "{{ patch_openstackversions | bool }}"
+          tags:
+            - setup
+
+        - name: Patch observabilityclient into openstackclient
+          ansible.builtin.shell:
+            cmd: |
+              oc exec openstackclient -- python3 -m ensurepip --upgrade
+              oc exec openstackclient -- python3 -m pip install --upgrade aodhclient
+              oc exec openstackclient -- python3 -m pip install --upgrade python-observabilityclient
+          when: "{{ patch_openstackversions | bool }}"
+          tags:
+            - setup
+
+        - name: Wait until the oscp is resolved the changes to continue
+          ansible.builtin.shell:
+            cmd: |
+              oc get oscp | grep "Setup complete"
+          retries: 24
+          timeout: 5
+          until: output.stdout_lines | length == 1
+          register: output
+          when: "{{ patch_openstackversions | bool }}"
+          tags:
+            - setup
```

Comment:
These are the setup steps needed for autoscaling; these tasks are required because we need a different version of heat and aodh than is deployed.

```suggestion
```

Reply:
Done